src/natives/screenshot/win/Program.cs

```diff
@@ -174,7 +174,7 @@ struct WindowBorderOnScreenFlags {
                     wi.rcClient.right -= 1;
                 }
             }
-            else if(browser == "ie") {
+            else if(browser == "iexplore") {
```

Comment:
The decision to use "iexplore" alias instead of "ie" is assosiated with [this](https://github.com/DevExpress/testcafe/issues/207) ?